client/src/components/MultiJurisdictionAudit/CSVForm/index.tsx

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { Formik, FormikProps, Form } from 'formik'
+import styled from 'styled-components'
+import { HTMLSelect, FileInput, H4 } from '@blueprintjs/core'
+import FormWrapper from '../../Atoms/Form/FormWrapper'
+import FormButton from '../../Atoms/Form/FormButton'
+import schema from './schema'
+import { ErrorLabel } from '../../Atoms/Form/_helpers'
+import FormSection, {
+  FormSectionDescription,
+} from '../../Atoms/Form/FormSection'
+import { IFileInfo } from '../useJurisdictions'
+
+export const Select = styled(HTMLSelect)`
+  margin-top: 5px;
+`
+
+interface IValues {
+  csv: File | null
+}
+
+interface IProps {
+  csvFile: IFileInfo
+  uploadCSVFile: (csv: File) => Promise<boolean>
+  deleteCSVFile: () => Promise<boolean>
+  filePurpose: 'ballot-manifest' | 'batch-tallies'
+  enabled: boolean
+}
+
+const CSVFile: React.FC<IProps> = (props: IProps) => {
+  // Force the form to reset every time props.csvFile changes
+  // E.g. if we upload or delete a file
+  // See https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recap
+  return <CSVFileForm key={Date.now()} {...props} />
+}
+
+const CSVFileForm = ({
+  csvFile,
+  uploadCSVFile,
+  deleteCSVFile,
+  filePurpose,
+  enabled,
+}: IProps) => {
+  const { file, processing } = csvFile
+  const isProcessing = !!(processing && !processing.completedAt)
+  const [isEditing, setIsEditing] = useState<boolean>(!file || isProcessing)
+
+  return (
+    <Formik
+      initialValues={{ csv: null }}
+      validationSchema={schema}
+      onSubmit={async (values: IValues) => {
+        if (values.csv) {
+          uploadCSVFile(values.csv)
+        }
+      }}
+    >
+      {({
+        handleSubmit,
+        setFieldValue,
+        values,
+        touched,
+        errors,
+        handleBlur,
+      }: FormikProps<IValues>) => (
+        <Form>
+          <FormWrapper>
+            {filePurpose === 'ballot-manifest' ? (
+              <FormSection>
+                <H4>Ballot Manifest</H4>
+                <FormSectionDescription>
+                  Click &quot;Browse&quot; to choose the appropriate Ballot
+                  Manifest file from your computer. This file should be a
+                  comma-separated list of all the ballot boxes/containers used
+                  to store ballots for this particular election, plus a count of
+                  how many ballot cards (individual pieces of paper) are stored
+                  in each container.
+                  <br />
+                  <br />
+                  <a
+                    href={`${window.location.origin}/sample_ballot_manifest.csv`}
```

Comment:
why do we need `window.location.origin` here? Won't just `/sample_ballot_manifest.csv` do?

Reply:
Huh, good catch. Not sure why that got put in there, haha.